components/lib/VHeader.vue

```diff
@@ -162,9 +162,9 @@ header {
 
 @media (--viewport-medium) {
   .header-menu {
-    position: fixed;
+    position: absolute;
     max-height: 0;
-    top: 4.8rem;
+    top: 10rem;
```

Comment:
Here the positioning of the header menu is relative to the document body and any changes in the header height will require changing this value. For example, if we remove the top banner later, it will break the menu positioning.

I suggest we make:
- the wrapping header tag `relative`,
- position the header menu to 100% from top relative to the header, and
- change the max height when the menu is open to `100vh` instead of `100%`.

```css
header {
  position: relative;  // <--
  ...
}

@media (--viewport-medium) {
  .header-menu {
    position: absolute; // <--
    max-height: 0;
    top: 100%;
    left: 0;
    right: 0;
    ...
  }
}

.header-menu.show {
  max-height: 100vh;  // <--
  padding: 1rem;
}
```

Reply:
@ajithranka on it.

Reply:
@ajithranka made the suggested change

## screenshot

With Banner:
<img width="329" alt="Screenshot 2022-02-11 at 12 34 21 PM" src="https://user-images.githubusercontent.com/40512676/153550057-0d4e7205-bddf-46d8-9a56-2667f0ac8043.png">

Without Banner:
<img width="329" alt="Screenshot 2022-02-11 at 12 34 33 PM" src="https://user-images.githubusercontent.com/40512676/153550081-7654ae27-4c4e-4e73-81d6-bc22bd5be8aa.png">